### ISSUE TITLE
When not on x86 `pub use __impl_type as __impl_type;` does not compile

### ISSUE THREAD

**Issue by IceTDrinker** [sarah-ek/pulp#11]
```console
error[E0432]: unresolved import `__impl_type`
   --> /home/ubuntu/.cargo/registry/src/index.crates.io-6f17d22bba15001f/pulp-0.19.6/src/core_arch/mod.rs:328:9
    |
328 | pub use __impl_type as __impl_type;
    |         ^^^^^^^^^^^^^^^^^^^^^^^^^^ no external crate `__impl_type`
```

the __impl_type only exists on x86 and likely is missing the same feature gate as the definition of the macro :)

**Comment by sarah-quinones** [comment]
fixed in 0.19.7

**Comment by IceTDrinker** [comment]
thanks a lot !
